clang/lib/APINotes/APINotesReader.cpp

```diff
@@ -527,6 +527,13 @@ class TagTableInfo
       Info.EnumExtensibility =
           static_cast<EnumExtensibilityKind>((Payload & 0x3) - 1);
 
+    uint8_t Copyable =
+        endian::readNext<uint8_t, llvm::endianness::little>(Data);
+    if (Copyable == 1)
+      Info.setSwiftCopyable(std::optional(false));
+    else if (Copyable == 2)
+      Info.setSwiftCopyable(std::optional(true));
```

Comment:
Can we use named constants for the non-traditional boolean values? Perhaps `kSwiftNonCopyable` and `kSwiftCopyable`? This should ensure that we do not drift.

Reply:
Good point! Added the constants.